bootstrap_cfn/ec2.py

```diff
@@ -36,9 +36,13 @@ def get_instance_private_ips(self, instance_id_list):
     def set_instance_tags(self, instance_ids, tags={}):
         return self.conn_ec2.create_tags(instance_ids, tags)
 
-    def create_sg(self, name):
+    def create_sg(self, name, vpc_id):
         return self.conn_ec2.create_security_group(
-            name, 'bootstrap generated SG')
+            name, 'bootstrap generated SG', vpc_id)
+
+    def delete_sg(self, name):
+        return self.conn_ec2.delete_security_group(
+            name, 'bootstrap deleted SG')
```

Comment:
Two further questions:

1) Why does delete have one too?
2) Is it possible to have a better indicator of what the SG actually is/does/what purpose it serves?